plugin/SqueezeESP32/Graphics.pm

```diff
@@ -244,6 +244,11 @@ my @extra = (
 		bar => 1, secs => -1,  width => $width,
 		# extra parameters (bars)
 		params => [$VISUALIZER_SPECTRUM_ANALYZER_ESP32, $width_low, int ($width/$spectrum->{full}->{band}), $spectrum->{scale}] },	
+		# mode E10
+		{ desc => ['VISUALIZER_SPECTRUM_ANALYZER', 'PLUGIN_SQUEEZEESP32_TAUDIO'],
+		bar => 1, secs => -1,  width => $width,
+		# extra parameters (19 bars for led vu)
+		params => [$VISUALIZER_SPECTRUM_ANALYZER_ESP32, $width_low, 19, $spectrum->{scale}] },	
```

Comment:
I'm not sure I understand the use of 19 here and how it does or does not conflict with other options.

Reply:
This is a specific force for the device to run the same number of visu bars as LEDs for a specific lev_vu display mode.
With all the customization done for different display sizes, the visualization mode and style selection was too risky to tackle, and I didn't want to have the overhead of running an additional vu_update with different parameters (as Chuck is on the SqueezeAMPToo code).  
As a simple solution I select the visualizer mode based on the number of bars (and style if used).  
I did try to enable the separate Visualizer Selection menu that is used on the transporter on screen2, but failed and gave up.